src/1-Profile/Edit-Profile.tsx

```diff
@@ -203,7 +232,23 @@ floatingEditIcon: {
   width: 35,
   height: 35
 },
-
+backButton: {
+  //position: "absolute",
```

Comment:
Let's clean up these comments.
Also, `backButton` will be on nearly every page, can we abstract the styling and maybe implementation too?